### DISCUSSION TITLE
F-droid

### DISCUSSION THREAD

**Discussion by Belpol** [ntherning/StorytelReaderMods#8] [Q&A]
Everything worked until I got stuck at F-Droid. 
It installed and launched fine. But there are no apps visible. 
Upon refreshing the list I get the error: 'Error getting F-Droid index file - No mirrors available'. 

I rebooted multiple times already. 

Any ideas? Thanks!

**Comment by ntherning** [comment]
Hi, sounds to me like it could have been a temporary network error? Either on your end or on the F-Droid server side. Did you try the next day? Also make sure you reader is connected to the internet.

**Comment by raymon-m** [comment]
I had this same problem. What fixed it for me is a hint that I got from one of the issues posted: installing fdroid with the apk for the opposite generation of devices than the one you have. So if you have a gen2 device, use the gen1 installation instructions and vice versa. Hope this helps!

**Reply by ntherning** [reply]
#12 has been merged which fixes the README.
